### PR TITLE
Added camera bag file to utilize in both perception pipeline demo and moveit_middleware_benchmark

### PR DESCRIPTION
**Pull request by CihatAltiparmak**
Hello, i wanted to add the bag file which it has some camera topics. I think it would be better to use this bag file for both perception pipeline demo and moveit_middleware_benchmark tool. Friendly tag here (@sjahr )

### CODE REVIEW

**Review by sjahr — APPROVED** [review]
Thanks

**Review by sjahr — COMMENTED** [review]
On second thought, do you mind putting the bagfile into a new folder called something like `rosbags`, `bagfiles` etc? I think that would be cleaner. Because I'd probably do that: Don't forget to add that folder to the CMakeLists.txt install command :wink:

**Comment by CihatAltiparmak** [comment]
Hello,
I've applied your suggestion and tested in perception pipeline demo (it works). But just to validate, it would be super great if you test with perception pipeline. (unnecessary datas, some broken stuffs and so on)

Btw i've fixed the typo in author :wink: :cool:

File: moveit_benchmark_resources/package.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <version>1.0.0</version>
   <description>Databases for MoveIt benchmarking</description>
```
Comment (sjahr) [inline moveit_benchmark_resources/package.xml]:
```suggestion
  <description>Databases and bag files for MoveIt 2 benchmarking and tutorials</description>
```

File: moveit_benchmark_resources/package.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 
-  <author email="sebastian.jahr@picknik.ai">Ioan Sucan</author>
+  <author email="sebastian.jahr@picknik.ai">Sebastian Jahr</author>
```
Comment (sjahr) [inline moveit_benchmark_resources/package.xml:7]:
```suggestion
  <author email="sebastian.jahr@picknik.ai">Sebastian Jahr</author>
  <author email="cihataltiparmak1@gmail.com">Cihat Altıparmak</author>
```

**Review by sjahr — APPROVED** [review]

File: moveit_benchmark_resources/package.xml (hunk 2 above)
Comment (CihatAltiparmak) [reply moveit_benchmark_resources/package.xml:7]: Ohh my god, thank you so much, but is it really necessary?

Comment (sjahr) [reply moveit_benchmark_resources/package.xml:7]: If you don't mind :+1: The bag files will probably make up half of what is in this package so I think we can add you here

Comment (CihatAltiparmak) [reply moveit_benchmark_resources/package.xml:7]: Thank you:)

**Pull request merged**